public/index.php

```diff
@@ -20,8 +20,20 @@
     unset($path);
 }
 
+// Set default timezone if not available in php.ini
+if (!ini_get('date.timezone')) {
+    date_default_timezone_set('UTC');
+}
```

Comment:
The framework itself should **NOT** adjust PHP globals.

This should come from your PHP settings, which are to be defined by you when building your production-ready docker containers (or whichever mechanism you use to provision production)